### PR TITLE
Remove travis build stages

### PR DESCRIPTION
**Pull request by alissapajer**

### CODE REVIEW

File: src/main/g8/.travis.yml [hunk 1]
```diff
@@ -27,22 +21,16 @@ env:
...
-
-      git:
-        depth: false
+script:
+    - set -e
+
+    - \$SBT ++\$TRAVIS_SCALA_VERSION test
+
+    - |-
+      if [ $TRAVIS_PULL_REQUEST == "false" ] && [[ "$TRAVIS_BRANCH" =~ backport/v.*|master ]]; then
```
Comment (djspiewak) [inline src/main/g8/.travis.yml]: The `TRAVIS_PULL_REQUEST` stuff needs backslashes.

Comment (alissapajer) [reply src/main/g8/.travis.yml]: I copied this line directly from [here](https://github.com/slamdata/quasar/blob/master/.travis.yml#L43)

Comment (alissapajer) [reply src/main/g8/.travis.yml]: Is there something different about this situation?

Comment (djspiewak) [reply src/main/g8/.travis.yml]: Yep. But you aren't writing a `.travis.yml` file. You're writing a *template* which will be instantiated into the `.travis.yml` file. The `$` character has special semantics within the template. Note the surrounding lines also escape the `$`.

Comment (alissapajer) [reply src/main/g8/.travis.yml]: @djspiewak to save on iterations, does this look right?
```
    - |-
      if [ \$TRAVIS_PULL_REQUEST == "false" ] && [[ "$TRAVIS_BRANCH" =~ backport/v.*|master ]]; then
        \$SBT transferPublishAndTagResources;
        TRAVIS_JOB_NUMBER=1 scripts/publishAndTag 'slamdata/$name;format="lower,hyphen"$';
      fi
```
What are the escape rules for `"$TRAVIS_BRANCH"`; does that need to be escaped too?

Comment (djspiewak) [reply src/main/g8/.travis.yml]: It does. The templating stuff is pretty dumb

**Comment by alissapajer** [comment]
@djspiewak pushed the escapes.

**Pull request merged**